recaf-ui/src/main/java/me/coley/recaf/ui/plugin/InstalledTab.java

```diff
@@ -0,0 +1,42 @@
+package me.coley.recaf.ui.plugin;
+
+import dev.xdark.recaf.plugin.Plugin;
+import dev.xdark.recaf.plugin.PluginContainer;
+import dev.xdark.recaf.plugin.PluginInformation;
+import dev.xdark.recaf.plugin.RecafPlugin;
+import javafx.scene.control.Tab;
+import me.coley.recaf.ui.plugin.item.InstalledPluginItem;
+import me.coley.recaf.util.logging.Logging;
+import org.slf4j.Logger;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author xtherk
+ */
+public class InstalledTab extends Tab {
+    private final Logger logger = Logging.get(this.getClass());
```

Comment:
Can be `static` and initialized like other loggers.

Reply:
ok